codespell_lib/data/dictionary.txt

```diff
@@ -20292,6 +20327,7 @@ keywoards->keywords
 keywork->keyword
 keyworkd->keyword
 keyworkds->keywords
+keyworks->keywords
```

Comment:
Perhaps `keywork->keyword, key work,` or far-fetched?

Reply:
Hmm, yes. I think we could add it in together with `keyworks->key works`. Sounds like something you would see in a scientific paper, for example. Now done.